### PR TITLE
Fix card styles

### PR DESCRIPTION
**Pull request by nikitakukharchuk**
Fix styles for title

### CODE REVIEW

File: src/components/cards/Cards.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ type PropsCardsType = {
 };
 
 export const Cards: FC<PropsCardsType> = ({ cards }) => (
+    (cards)
```
Comment (macroserP) [inline src/components/cards/Cards.tsx]: can you apply prettier and rename s to styles?

File: src/components/cards/Cards.tsx [hunk 2]
```diff
@@ -19,4 +21,6 @@ export const Cards: FC<PropsCardsType> = ({ cards }) => (
       />
     ))}
   </div>
+        :
```
Comment (gitkoshelew) [inline src/components/cards/Cards.tsx]: Fix formatting, please

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/components/cards/Cards.tsx (hunk 2 above)
Comment (nikitakukharchuk) [reply src/components/cards/Cards.tsx]: https://github.com/gitkoshelew/CC.ui/pull/54/commits/933e4da7d9c6a8e34aff28fda08bde1f0c7aac74
fix formatting as you wrote earlier

**Review by macroserP — DISMISSED** [review]

**Review by visiteur85 — DISMISSED** [review]

File: src/components/cards/Cards.tsx (hunk 1 above)
Comment (nikitakukharchuk) [reply src/components/cards/Cards.tsx]: rename s to styles and apply prettier

**Review by macroserP — DISMISSED** [review]

**Review by gitkoshelew — APPROVED** [review]

**Review by macroserP — APPROVED** [review]

**Pull request merged**